ui/ceval/src/stockfishProtocol.js

```diff
@@ -1,22 +1,25 @@
 var m = require('mithril');
 
+var EVAL_REGEX = new RegExp(''
+  + /^info depth (\d+) seldepth \d+ multipv (\d+) /.source
+  + /score (cp|mate) ([-\d]+) /.source
+  + /(?:(upper|lower)bound )?nodes (\d+) nps (\d+) /.source
+  + /(?:hashfull \d+ )?tbhits \d+ time (\d+) /.source
+  + /pv(.*)/.source);
+
 module.exports = function(worker, opts) {
 
   var work = null;
   var state = null;
-  var minLegalMoves = 0;
-  var startedAt = null;
 
   var stopped = m.deferred();
   stopped.resolve(true);
 
   var emit = function() {
     if (!work || !state) return;
-    minLegalMoves = Math.max(minLegalMoves, state.eval.pvs.length);
-    if (state.eval.pvs.length < minLegalMoves) return;
+    clearTimeout(state.emitTimer);
```

Comment:
How about using a new variable instead of a property on the publicly emitted object for `emitTimer`?